package.json

```diff
@@ -87,5 +88,12 @@
   "bugs": {
     "url": "https://github.com/xmldom/xmldom/issues"
   },
-  "license": "(LGPL-2.0 OR MIT)"
+  "license": "(LGPL-2.0 OR MIT)",
+  "auto-changelog": {
+    "commitLimit": false,
+    "remote": "upstream",
+    "startingVersion": "0.3.0",
```

Comment:
Wouldn't we have to update this after every release?

Reply:
Without adding this `auto-changelog` failed with an error with a not very helpful message, I assume because of all the inconsistencies in earlier releases/tags, etc. Since we don't care how compatible it is to those older releases I just added this to be able to use the tool.

But I just checked again: since the current approach is to just dump the information about the unreleased changes, it' possible to remove this option.